### PR TITLE
Update versions of GraphQL Java, Guava and JS/ES compilation target

### PR DESCRIPTION
**Pull request by pcdavid**
Move to the most recent versions of GraphQL Java and Guava, and target an earlier version of JS/ES (ES2019 instead of ES2020) to improve compatibility.

### CODE REVIEW

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
Everything seems fine but an important issue has been discovered with GraphQL Java 16, could you update to 16.1 please?

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
sirius-web-graphql-utils is still using GraphQL Java 14

File: backend/sirius-web-graphql-utils/pom.xml [hunk 1]
```diff
@@ -47,7 +48,7 @@
 		<dependency>
         	<groupId>com.graphql-java</groupId>
         	<artifactId>graphql-java</artifactId>
-        	<version>14.0</version>
+        	<version>${graphql-java.version}</version>
```
Comment (sbegaudeau) [inline backend/sirius-web-graphql-utils/pom.xml:51]: Could you fix the problem in the appropriate commit please? Everything is fine apart from that.

**Review by sbegaudeau — CHANGES_REQUESTED** [review]

File: backend/sirius-web-graphql-utils/pom.xml (hunk 1 above)
Comment (pcdavid) [reply backend/sirius-web-graphql-utils/pom.xml:51]: I've squashed all the GraphQL updated into a single commit (14.0 => 16.1).

**Review by sbegaudeau — APPROVED** [review]

**Pull request merged**